Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/render.ads

```diff
@@ -0,0 +1,11 @@
+with System;
+
+with GL.Objects.Programs;
+
+with Glfw;
+
+package Render is
```

Comment:
OK, I'll follow your advice.
Either way the procedure would be better entitled Render_Dot.
This horrible structure evolved from my attempts to emulate SuperBible's "framework" structure which initially confused me greatly.
So I decided to get familiar with GLFW by working through Anton's OpenGL 4 Tutorials (https://capnramses.github.io//opengl/index.html) the result of which I intend to supply as my next example.
Having worked through these tutorials I than had a sort of framework which I then adapted (rather badly) to suit the SuperBible framework.
With regard to the SuperBible framework, the idea is to have a fairly fixed structure to which, for any example, only two procedures need be supplied: Startup and Render.
This is what I have been trying to emulate with run.adb in which I now include procedure Set_Up (see below) rather than Startup and now intend to include Render_<Example_Name>.
After much thinking I've also decided to change the OpenGL objects such as buffers and shader program variables into package variables rather than to declare them in Main_Loop and pass them as procedure parameters. This is because, for more complex examples, the number OpenGL objects becomes quite large.
OH! I've just noticed that, in subsequent examples, I moved Startup (renamed Set_Up) from the Initialization (framework) package into run.adb but haven't done it for Just_A_Dot yet but will do so next (Wish now that I had done it sooner but I am a master of bad decisions!).
Then run.adb will contain Main_Loop, Set_Up and  Render_<Example_Name>.
This seems to me a reasonable way of setting up a SuperBible "framework" requiring minimal changes for different examples. So that, hopefully, the different examples can be generated by only changing Set_Up and  Render_<Example_Name> (though, I expect a small number of other changes may be needed).
By the way SuperBible's Startup procedure, in my opinion, is horrendous!
No doubt you will recommend welcome improvements to these ideas.
I hope this isn't just making more of a mess!